### ISSUE TITLE
Aishell testing data benchmark

### ISSUE THREAD

**Issue by misbullah** [PaddlePaddle/DeepSpeech#118]
Hi, 

Today, I did evaluation using different models (AM and LM). Plese see the following table.

![image](https://user-images.githubusercontent.com/385009/34866348-48658200-f7b7-11e7-930a-a80fe8cb225b.png)

I reduce sampling rate to be 8000 Hz when doing test for BaiduCN1.2k model because I think the model was trained using 8000 Hz dataset.

Question:
1. Why I get very worse result for BaiduCN1.2K? (As we know the model was trained using larger training data than AISHELL and the model parameter also larger than AISHELL model?
2. How to use tune.py to find good tuning parameter for alpha and beta?
   I cannot find any examples for the python code?

Thanks,
Alim

**Comment by pkuyym** [comment]
For question 1:
> Why I get very worse result for BaiduCN1.2K? (As we know the model was trained using larger training data than AISHELL and the model parameter also larger than AISHELL model?
There are some key potential reasons:
1) the two vocabulary of AiShell and BaiduCN1.2k are not well normalized, like (一 and '1' should be same).
2) the training data are from different domains, BaiduCN1.2k consists of video audios while audios of Aishell are more general. 

For question 2:
1) Please refer to https://github.com/PaddlePaddle/DeepSpeech/blob/develop/examples/librispeech/run_tune.sh
2) You can tune alpha and beta in a coarse range with less data (like two batches) first, then tine the values with more data finely.

**Comment by misbullah** [comment]
Hi @pkuyym,

It is very clear explanation!

Thanks a lot.

Best,
Alim

**Comment by zhaoningning** [comment]
@misbullah  I tuned alpha and beta  and didn't find any improvment  contrast to greddy decoding, can you share your hyper parameters to me? Besides,I use beam_width=10 and cutoff_top_n=5,the decoding speed is very slow,do you have any ideas for that? thx!

**Comment by misbullah** [comment]
Hi @zhaoningning,

I don't tune any hyper-parameter. I only use the similar configuration from AISHELL example. 
The decoding speed is depend on the mechanism that you are using either GPU or CPU. 

Which model do you use now?

Thanks.

**Comment by DimaMcar** [comment]
Why didn't you use larger LM (Mandarin LM Large)?
